x-pack/plugins/timelines/public/container/use_update_alerts.ts

```diff
@@ -37,7 +37,7 @@ export const useUpdateAlertsStatus = (
   const { http } = useKibana<CoreStart>().services;
   return {
     updateAlertStatus: async ({ status, index, query }) => {
-      if (['detections-page', 'detections-rules-details-page'].includes(timelineId)) {
+      if (useDetectionEngine) {
```

Comment:
May want to rename -- wasn't immediately clear how the detection engine wasn't involved with updating alert status, but clicking through to see the conditional showed this is just a feature flag check between legacy impl and RAC RBAC routes. I guess this makes sense since the route ends up not being a DE route, but RAC, so don't mind me....

Do need some JSDocs though, haha 😉 

<p align="center">
  <img width=500" src="https://user-images.githubusercontent.com/2946766/136099168-4857275b-6b2e-4bc9-aa9b-9fc0abf855dc.png" />
</p>